bundle/lib/source/DobbyRootfs.cpp

```diff
@@ -645,6 +646,13 @@ bool DobbyRootfs::constructRootfs(int dirfd,
         return false;
     }
 
+    // write empty /etc/resolv.conf file in case we want to mount it from the host
+    if (openat(dirfd, "etc/resolv.conf", O_CREAT | O_WRONLY | O_TRUNC) < 0)
```

Comment:
you need to add the mode (0700 etc) as the last argument in openat() call otherwise compiler on Xi6 throws an error. See https://github.com/rdkcentral/Dobby/commit/586a0dbe5932a0ce7709730fdede44e97f823021